template/experiment-server/webserver.py

```diff
@@ -22,10 +23,14 @@ def default() -> RedirectResponse:
 
 
 @app.get("/experiment")
-def serve_frontend() -> FileResponse:
+def serve_frontend(request: Request, PROLIFIC_PID: str | None = "", STUDY_ID: str | None = "", SESSION_ID: str | None = "") -> FileResponse:
     """
     Serves static UI
     """
+    request.session["PROLIFIC_PID"] = PROLIFIC_PID
```

Comment:
Should these variables be lowercase?

Reply:
that is how the querystring parameters are defined... theoretically, sure. urls not being case sensitive, but I was just matching what Prolific calls out.

Reply:
just did a test and it does not work. to be clear, if the querystring has them capitalized or lower case it works as long as the function parameters are capitalized, but if the querystring parameters are capitals and the parameter names in the function header are lower case it does not work and Prolific will send them capitalized.